support-frontend/assets/components/layout/container.tsx

```diff
@@ -12,15 +13,15 @@ type ContainerElement =
 	| 'nav'
 	| 'section';
 
-type Props = {
+interface ContainerProps extends HTMLAttributes<HTMLElement> {
```

Comment:
So this gives us the ability to pass any additional html attribute to the Container without having to explicitly add it to the prop types?

Reply:
Yes, it just makes it easier to add things like an ID if necessary.